cli/commands.go

```diff
@@ -71,15 +72,16 @@ func Commands(metaPtr *meta.Meta) map[string]cli.CommandFactory {
 					"ldap":     credLdap.Factory,
 				},
 				LogicalBackends: map[string]logical.Factory{
-					"aws":        aws.Factory,
-					"consul":     consul.Factory,
-					"postgresql": postgresql.Factory,
-					"cassandra":  cassandra.Factory,
-					"pki":        pki.Factory,
-					"transit":    transit.Factory,
-					"mssql":      mssql.Factory,
-					"mysql":      mysql.Factory,
-					"ssh":        ssh.Factory,
+					"aws":             aws.Factory,
+					"consul":          consul.Factory,
+					"postgresql":      postgresql.Factory,
+					"cassandra":       cassandra.Factory,
+					"pki":             pki.Factory,
+					"transit":         transit.Factory,
+					"mssql":           mssql.Factory,
```

Comment:
ugh my bad :D yes. Thats ok. For the mssql one i'd still recommend using azuresql